src/modules/claims/claims.service.ts

```diff
@@ -97,18 +103,22 @@ const {
 export class ClaimsService {
   private _claimManager: string;
   private _claimManagerInterface = ClaimManager__factory.createInterface();
-  private _vcIssuerVerifier: VCIssuerVerification;
   private _claimRevocation: ClaimRevocation;
-
+  private _vcIssuerVerifier: VCIssuerVerification;
+  protected verifyProof: (
```

Comment:
Why `protected` over `private`? I tend to think that one should use `private` unless `protected` is a part of the class design (which it might be in this case, and I'm just not seeing it yet 😄 )